src/components/Comments.jsx

```diff
@@ -9,14 +9,18 @@ const CommentSection = () => {
     const  {article_id} = useParams();
 
```

Comment:
the loading state should go here - the loading component sould only render if the state gets set to false when the request fulfills. please add this